what_is.md

```diff
@@ -15,16 +15,189 @@ kernelspec:
+the basis for a powerful [framework for statistical calculation](sec_what_is_analysis).
+But first we shall see how the ancestry can be used as an efficient
+*evolutionary encoding* for DNA sequences.
+
+(sec_what_is_dna_data)=
+
+## An efficient encoding of DNA data
+
+A tree sequence can be used to describe patterns of genetic variation by combining the
+trees with a knowledge of where *mutations* occur on their branches. Here's how that
+might look in our simple example:
+
+```{code-cell} ipython3
+seed = 3
+mutated_ts = msprime.sim_mutations(ts, rate=1e-7, random_seed=seed)
+
+mut_labels = {}  # An array of labels for the mutations, listing position & allele change
+l = "{:g} ({}→{})"
+for mut in mutated_ts.mutations():  # This entire loop is just to make pretty labels
+    site = mutated_ts.site(mut.site)
+    older_mut = mut.parent >= 0  # is there an older mutation at the same position?
+    prev = mutated_ts.mutation(mut.parent).derived_state if older_mut else site.ancestral_state
+    mut_labels[mut.id] = l.format(site.position, prev, mut.derived_state)
+
+SVG(mutated_ts.draw_svg(size=sz, node_labels=labels, mutation_labels=mut_labels, style=style1))
+```
+
+The trees tell us that, for example, the final mutation (at position 980) is inherited
+by genomes $\mathrm{a}$ to $\mathrm{i}$. These genomes must have a *G* at that position,
+compared to the original value of *C*, which only genome $\mathrm{j}$ has retained. In
+other words, the combination of ancestry-plus-mutations fully defines the genetic
+variation at all 11 variable sites. The variant at each of these 11 sites can be easily
+extracted from the tree sequence:
+
+```{code-cell} ipython3
+haplotypes = mutated_ts.haplotypes()
+print("\n".join(sorted([f"Genome {labels[i]}: {h}" for i, h in enumerate(haplotypes)])))
+```
+
+This is only a tiny example; real-world use cases may involve many millions of genomes
+with hundreds of thousands of trees, spanning chromosomes of hundreds of megabases
+in length. Tree sequences scale efficiently to such huge datasets because of one key
+property: adjacent trees along the genome are highly correlated, that is, they
+*share structure*. In our example, this becomes evident if we highlight the branches,
+or "edges" in tree sequence terminology, that remain unchanged between the first
+and the second tree (more visualization possibilities are detailed in the
+[visualization tutorial](sec_tskit_viz)).
+
+```{code-cell} ipython3
+kept_edges = [e for e in ts.edges() if e.left==0 and e.right>ts.breakpoints(True)[1]]
+style3 = (
+    ",".join(f"#svg1 .tree:not(.t2) .node.a{e.parent}.n{e.child} > .edge" for e in kept_edges)
+    + "{stroke:#00DD00; stroke-width: 2px}"
+    + style1)
+SVG(ts.draw_svg(size=sz, root_svg_attributes={'id':'svg1'}, node_labels=labels, style=style3))
+```
+
+Another way to think about shared structure is to notice that the second tree can be
```

Comment:
Yep, reading back, I think we don't need SPRs in here, although it would be good to link to a tutorial where they are explained. Having an animation would be nice, but too much work at the moment, I think (especially as it helps with the animation to have the recombination nodes, which we don't have for this example)